lib/openid_token_proxy.rb

```diff
@@ -1,5 +1,9 @@
 require File.expand_path('../../config/initializers/inflections', __FILE__)
 
+require 'openid_connect'
+
+require 'openid_token_proxy/error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.